### DISCUSSION TITLE
Push error

### DISCUSSION THREAD

**Discussion by TarekMohammmed9** [community/community#30604] [New to GitHub]
I get this error when i use git push and use my github passowrd any recommendations ??

remote: Support for password authentication was removed on August 13, 202
1.                                                                       
remote: Please see https://docs.github.com/en/get-started/getting-started
-with-git/about-remote-repositories#cloning-with-https-urls for informati
on on currently recommended modes of authentication.                     
fatal: Authentication failed for 'https://github.com/TarekMohammmed9/alx-
pre_course.git/'

**Comment by melvincwng** [comment]
@TarekMohammmed9 Hi Tarek, try these solutions [here](https://stackoverflow.com/questions/68781928/support-for-password-authentication-was-removed-on-august-13-2021) & see if they work.

Hope this answers your question.

Credits: StackOverFlow

**Reply by TarekMohammmed9** [reply]
same errors and many other errors 
can't find a solution for this error 
and i don't know what to do

**Reply by Tofiyinkay** [reply]
have you found a solution to this error? I'm having this same issue

**Comment by airtower-luna** [comment]
You cannot use your password for `git push` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).

**Comment by JohnSmithW** [comment]
In Github, go to Settings -> Developer settings(At the bottom of left sidebar) -> Personal access tokens (Left sidebar) -> Generate new token, after you generate it, save it somewhere on your PC and use it as a password whenever you need to authorize via Github.
